src/components/ExchangeInput.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChangeEventHandler } from 'react'
+import { FormControl, InputAdornment, InputLabel, OutlinedInput } from '@mui/material'
+
+interface ExchangeInputProps {
+	icon: React.ReactNode
+	inputProps?: { [key: string]: string | number | boolean | null }
```

Comment:
This definition is not type-safe. Pls use the proper interface (`InputBaseComponentProps`)